core/vm/contracts.go

```diff
@@ -155,7 +155,7 @@ var PrecompiledContractsLuban = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{9}): &blake2F{},
 
 	common.BytesToAddress([]byte{100}): &tmHeaderValidate{},
-	common.BytesToAddress([]byte{101}): &iavlMerkleProofValidatePlanck{},
+	common.BytesToAddress([]byte{101}): &iavlMerkleProofValidateLuban{},
```

Comment:
should not modify `PrecompiledContractsLuban`, 
need to create a new `PrecompiledContractsPlato`